deepchem/feat/graph_data.py

```diff
@@ -274,11 +321,14 @@ def __init__(self, graph_list: Sequence[GraphData]):
             batch_node_pos_features = None
 
         # create new edge index
+        # number of nodes in each graph
         num_nodes_list = [graph.num_nodes for graph in graph_list]
+        # cumulative number of nodes for each graph
```

Comment:
Can you raise an issue documenting this bug? I am not sure I understand fully what the issue is, and want to make sure I can scrutinize it properly. We should also separate this bugfix out into a separate PR to track. It's not good practice to mix feature addition and bugfix in the same PR since it makes it hard to disentangle